binance/fee.go

```diff
@@ -0,0 +1,24 @@
+package binance
+
+import (
+	"strings"
+
+	"github.com/rafaelbmateus/binance-bot/config"
+)
+
+// Fee check trading fee rate.
+func Fee(symbol string) float64 {
+	if strings.ToUpper(symbol) == "BNB" {
+		return 0.075
+	}
+
+	return 0.1
+}
+
+func ProfitPerc(trade config.Trade) float64 {
+	return (trade.SellPrice - trade.BuyPrice) / (trade.SellPrice) * 100
+}
+
+func Profit(trade config.Trade) float64 {
+	return (Fee(trade.BuyWith()) / 100) * (trade.BuyPrice + trade.SellPrice)
```

Comment:
I'm just thinking about this calc, if i bought some symbol for USDT 100 and sold it for 110, i should have 10% of profit, however the calc results in a different result:
```
(Fee(trade.BuyWith()) / 100) * (trade.BuyPrice + trade.SellPrice)
(0,1 / 100) * (100 + 110)
(0,001) * (210)
0,21
```
There are more details here: https://www.binance.com/en/support/faq/e85d6e703b874674840122196b89780a

Reply:
That's right man. I'll fix it